### ISSUE TITLE
Consider adding CollectionCondition.texts(List<String>) and CollectionCondition.texts(List<String>)

### ISSUE THREAD

**Issue by yashaka** [codeborne/selenide#419]
Sometimes "texts data to be asserted for collection" comes as `List<String>` not `String[]`

This leads to such ugly code:

```
headers().shouldHave(exactTexts(listOfTexts.toArray(new String[0])));
```

The idea is to add additional conditions which can accept `List<String>` instead of `String.../String[]`

so code from above becomes:

```
headers().shouldHave(exactTexts(listOfTexts)));
```

**Comment by asolntsev** [comment]
Agree. Let's use Lists!

**Comment by asolntsev** [comment]
@yashaka implemented by @pavelsmolensky in PR #425 
Merged to Selenide 4.1
